tests/EventTest.php

```diff
@@ -119,32 +119,27 @@ public function testToArray(): void
         $this->assertEquals($expected, $event->toArray());
     }
 
-    public function testToArrayWithMessage(): void
+    /**
+     * @dataProvider toArrayWithMessageDataProvider
+     */
+    public function testToArrayWithMessage(array $message, $expectedValue): void
     {
         $event = new Event();
-        $event->setMessage('foo bar');
+        call_user_func_array([$event, 'setMessage'], $message);
 
         $data = $event->toArray();
 
         $this->assertArrayHasKey('message', $data);
-        $this->assertEquals('foo bar', $data['message']);
+        $this->assertSame($expectedValue, $data['message']);
     }
 
-    public function testToArrayWithMessageWithParams(): void
+    public function toArrayWithMessageDataProvider(): array
     {
-        $expected = [
-            'message' => 'foo %s',
-            'params' => ['bar'],
-            'formatted' => 'foo bar',
+        return [
+            [['foo bar'], 'foo bar'],
+            [['foo %s', ['bar']], ['message' => 'foo %s', 'params' => ['bar'], 'formatted' => 'foo bar']],
```

Comment:
Please split this array into multiple lines (have a look at how other data providers are formatted)